_includes/partials/homePage-integrations.html

```diff
@@ -1,83 +1,217 @@
-            style="display: inline-block; width: fit-content;">View
-            Integrations in 3D</a>
+
+    <!-- Search Bar Container -->
+    <div class="sorting-options">
+        <select id="sort">
+            <option disabled selected value>Sort by</option>
+            <option value="asc">Alphabetically (A-Z)</option>
+            <option value="desc">Alphabetically (Z-A)</option>
+        </select>
+
+        <div id="search-container">
+            <input type="text" id="search" placeholder="Search..." />
+        </div>
+
+        <button class = "highlight" aria-label="Search">
+            Next
+        </button>
+    </div>
+
+    <div id="integContainer" class="integrations-container container"></div>
+
+    <div class="integrations-3d-link">
+        <div class="button-container">
+            <a class="highlight show-all-button toggle-text">Show More Integrations</a>
+            <a class="highlight show-all-button" href="/integrations">View Integrations in 3D</a>
+        </div>
     </div>
-</div>
 
-<script type="module">
-    // integrations.js
-    import data from '../../integrations/data.js';
+    <script type="module">
+        // integrations.js
+        import data from '../../integrations/data.js';
 
-    const integHeading = document.getElementById("integration-heading");
-    const roundedIntegNumber = Math.floor(data.length / 10) * 10;
-    integHeading.textContent = `${roundedIntegNumber}+ Built-In Integrations`;
+        const integHeading = document.getElementById("integration-heading");
+        const roundedIntegNumber = Math.floor(data.length / 10) * 10;
+        integHeading.textContent = `${ roundedIntegNumber } + Built - In Integrations`;
 
-    const integContainer = document.getElementById("integContainer");
+        const integContainer = document.getElementById("integContainer");
 
-    let allDataShown = false;
+        let allDataShown = false;
 
-    function renderIntegrationCards(data, showAll) {
-        const cardsToShow = showAll ? data.length : 40;
-        return data.slice(0, cardsToShow).map(item => {
-            const iconURL = item.color.split('/').slice(1).join('/');
+        function renderIntegrationCards(data, showAll) {
 
-            return `
-                <a href="${item.permalink}" class="integrations-card">
-                <div class="integrations-card-inner">
+            const cardsToShow = showAll ? data.length : 40;
+            return data
+                .slice(0, cardsToShow)
+                .map(
+                    (item) => `
+            <a href="${item.permalink}" class="integrations-card">
+              <div class="integrations-card-inner">
                 <div class="card-front">
-                <img src="${iconURL}" loading="lazy" id="logo-dark-light"
-                data-logo-for-dark="${iconURL}"
-                data-logo-for-light="${iconURL}" />
+                  <img src="${item.color}" loading="lazy" id="logo-dark-light"
```

Comment:
I see the error, you have to call the icon's url here in the image tag, how are you calling 'item.color' in <img> tag? this is causing the issue of images not getting displayed.